kbase-extension/scss/modules/_colours.scss

```diff
@@ -126,6 +118,20 @@ $palette: (
   "icon-type": rgb(0, 0, 0),
 );
 
+// Job status colours
+$job_status_colours: (
+  "created": map-get($kbase-palette, 'info'),
+  "estimating": map-get($kbase-palette, 'info'),
+  "queued": map-get($kbase-palette, 'info'),
+  "running": map-get($kbase-palette, 'info'),
+  "completed": map-get($kbase-palette, 'success-dark'),
+  "terminated": map-get($kbase-palette, 'warning-dark'),
+  "error": map-get($kbase-palette, 'error'),
+  "does_not_exist": map-get($kbase-palette, 'error'),
+);
+
+$palette: map-merge($kbase-palette, $job_status_colours);
```

Comment:
create a new palette using the job status colours and the existing kbase colours